ra/ra_test.go

```diff
@@ -1900,12 +1900,28 @@ func TestDeactivateRegistration(t *testing.T) {
 	_, _, ra, _, cleanUp := initAuthorities(t)
 	defer cleanUp()
 
-	err := ra.DeactivateRegistration(context.Background(), core.Registration{ID: 1})
-	test.AssertError(t, err, "DeactivateRegistration failed with a non-valid registration")
-	err = ra.DeactivateRegistration(context.Background(), core.Registration{ID: 1, Status: core.StatusDeactivated})
+	// Create a complete, deactivated registration
+	reg := &corepb.Registration{
+		Id:     1,
+		Status: string(core.StatusDeactivated),
+	}
```

Comment:
Super-nit: I'd move this to line 1912, immediately before it is used. Or even better, I'd inline the creation of two different `&corepb.Registration`s on lines 1914 and 1921 respectively.